libopenfpga/libfpgaaesencryption/src/AESfunctions.h

```diff
@@ -0,0 +1,17 @@
+#include <iostream>

```

Comment:
Not sure why <iostream> is used here. None of the functions in this file require a file stream. 
The include files for a header file should cover all the data structures used by the function declaration just in the file.

Reply:
You are correct, I did not notice that. I will remove it